packages/base/ApplicationUpdateNotification/src/ApplicationUpdateNotification/ApplicationUpdateNotification.tsx

```diff
@@ -62,22 +54,24 @@ export const ApplicationUpdateNotification = forwardRef<HTMLElement, Props>(
             variant='heading'
             size='medium'
             color='inherit'
-            className={classes.positionRelative}
+            className={'relative'}
```

Comment:
```suggestion
            className='relative'
```
For all the casesd like this (static strings in prop values), you can just leave them without the `{` and `}` (according to our style guides). I think the ESLint / prettier is supposed to auto-fix that, maybe it's not running or it became broken in our setup?

Reply:
I checked with Tomas and couldn't find any settings on ESLint regarding this topic. However, I should start remembering to stop adding the brackets.